public_dropin_notebook_environments/python39_notebook_gpu_tf/requirements-gpu.txt

```diff
@@ -6,4 +6,6 @@ openai==0.27.8
 tiktoken==0.4.0
 tensorflow==2.15.0
 transformers==4.36.0
-rouge-score==0.1.2
\ No newline at end of file
+rouge-score==0.1.2
```

Comment:
CC @lyapun as you have a PR that is adding a new GPU image

@akshoop I wonder if/can we have these updates in the notebooks repo as well?

It's my (maybe naive) understanding we should try and keep those images consistent with the ones here

Reply:
Could you point to me where in the notebooks repo that we should update to have these new dependencies?

Reply:
You got it - thanks.
Here's the directory:
https://github.com/datarobot/notebooks/tree/main/docker/kernels/python

And the most recent PR is a good example of making the changes:
https://github.com/datarobot/notebooks/pull/3389/files

Reply:
Thanks for the links. Let me know if notebooks repo changes should be made before my PR drop-in env changes merge or if it can be done afterwards in follow-up

Reply:
You got it - TBH I'm okay with either approach - @lyapun do you have a preference?